DrinkManagerWeb/Views/Admin/CreateUser.cshtml

```diff
@@ -0,0 +1,40 @@
+﻿@model DrinkManagerWeb.Models.ViewModels.UserViewModel
+@{
+    ViewBag.Title = "Administration - Users";
+    Layout = "_LayoutAdmin";
+}
+
+<h2>Create User</h2>
+<hr />
+<form method="post">
+    <div class="form-group row">
+        <label class="col-form-label text-md-right col-md-2" asp-for="Email"></label>
+        <div class="col-md-9">
+            <input asp-for="Email" class="form-control" />
+            <span asp-validation-for="Email" class="text-danger"></span>
+        </div>
+    </div>
+    <div class="form-group row">
+        <label class="col-form-label text-md-right col-md-2" asp-for="Password"></label>
+        <div class="col-md-9">
+            <input asp-for="Password" class="form-control" />
+            <span asp-validation-for="Password" class="text-danger"></span>
+        </div>
+    </div>
+    <div class="form-group row">
+        <label class="col-form-label text-md-right col-md-2" asp-for="ApplicationRoleId"></label>
+        <div class="col-md-9">
+            <select class="form-control" asp-for="ApplicationRoleId" asp-items="@Model.ApplicationRoles">
+                <option>Please select</option>
+            </select>
```

Comment:
Is it possible to set here "User" by default? Or it'd be even better to make this field required, because when you accidentally skip picking role, the user is still created without one and app refreshes Create view with all fields cleared instead of redirecting to Users view

Reply:
Fixed - choosing a role is now mandatory for creating and editing user.